extension/src/time_weighted_average/accessors.rs

```diff
@@ -54,3 +55,61 @@ fn time_weight_interpolated_average_accessor<'a>(
         }
     }
 }
+
+pg_type! {
+    #[derive(Debug)]
+    struct TimeWeightInterpolatedIntegralAccessor {
+        start : i64,
+        interval : i64,
+        prev : TimeWeightSummaryData,
+    pad : [u8;3],
+        unit : u32,
+        flags: u64,
+    next : TimeWeightSummaryData,
+    }
+}
+
+ron_inout_funcs!(TimeWeightInterpolatedIntegralAccessor);
+
+#[pg_extern(immutable, parallel_safe, name = "interpolated_integral")]
+fn time_weight_interpolated_integral_accessor<'a>(
+    start: crate::raw::TimestampTz,
+    interval: crate::raw::Interval,
+    prev: Option<TimeWeightSummary<'a>>,
+    next: Option<TimeWeightSummary<'a>>,
+    unit: String,
```

Comment:
This should have the same default to second behavior that the non-arrow version has.